src/reformat.c

```diff
@@ -616,6 +648,22 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image,
                     G = Y + Cb;
                     B = t - Cr;
                     R = t + Cr;
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+                } else if (state->mode == AVIF_REFORMAT_MODE_YCGCO_R) {
+#else
+                } else if (AVIF_FALSE) {
+#endif
+                    const int maxValue = (1 << (state->yuvDepth - 2)) - 1;
+                    const int16_t YY = unormY;
+                    const int16_t Cg = Cb * yuvMaxChannel;
+                    const int16_t Co = Cr * yuvMaxChannel;
+                    const int16_t t = YY - (Cg >> 1);
+                    G = AVIF_CLAMP(t + Cg, 0, maxValue);
+                    B = AVIF_CLAMP(t - (Co >> 1), 0, maxValue);
+                    R = AVIF_CLAMP(B + Co, 0, maxValue);
+                    G /= rgbMaxChannelF;
+                    B /= rgbMaxChannelF;
+                    R /= rgbMaxChannelF;
```

Comment:
General comment: Same here. This implementation is inefficient. It is fine for exporing the comression ratio of YCoCg-R, but we will need to redo the implementation when we start to care about speed.